src/domain/entities/jira-issue.ts

```diff
@@ -16,3 +16,7 @@ export type JiraIssue = {
 		readonly summary: string;
 	};
 };
+
+export const buildJiraIssueUrl = (baseUrl: string, issueKey: string) => {
```

Comment:
**note:** We could make this function a static method of `JiraIssue` but this would require converting `JiraIssue` to a class. I've left this small utility function here for now for simplicity.